commands/uploader.go

```diff
@@ -333,9 +364,9 @@ func (c *uploadContext) Await() {
 // given "endpoint". If no state has been explicitly set, an "unknown" state
 // will be returned instead.
 func getVerifyStateFor(endpoint lfsapi.Endpoint) verifyState {
-	key := strings.Join([]string{"lfs", endpoint.Url, "locksverify"}, ".")
+	uc := config.NewURLConfig(cfg.Git)
 
-	v, ok := cfg.Git.Get(key)
+	v, ok := uc.Get("lfs", endpoint.Url, "locksverify")
```

Comment:
I think this is where `hostsWithKnownLockingSupport` should be used. The way it is now, pushing to github won't warn about the lockVerify step. But it also won't halt a push if our locking services are down. Since our locking service is open to all repositories, it should assume that `locksVerify` is set to `"1"` instead of `""`.